src/util/Util.js

```diff
@@ -200,12 +200,12 @@ class Util extends null {
    */
   static escapeItalic(text) {
     let i = 0;
-    text = text.replace(/(?<=^|[^*])\*([^*]|\*\*|$)/g, (_, match) => {
+    text = text.replaceAll(/(?<=^|[^*])\*([^*]|\*\*|$)/g, (_, match) => {
       if (match === '**') return ++i % 2 ? `\\*${match}` : `${match}\\*`;
       return `\\*${match}`;
     });
     i = 0;
-    return text.replace(/(?<=^|[^_])_([^_]|__|$)/g, (_, match) => {
+    return text.replaceAll(/(?<=^|[^_])_([^_]|__|$)/g, (_, match) => {
```

Comment:
Unnecessary, please revert to `replace` as RegExp with global flag will go thru all instances.
```suggestion
    return text.replace(/(?<=^|[^_])_([^_]|__|$)/g, (_, match) => {
```